Control/TMSITable.cpp

```diff
@@ -819,13 +819,21 @@ vector< vector<string> > TMSITable::tmsiTabView(int verbosity, bool rawFlag, uns
 				}
 			} else if (header == "CREATED" || header == "ACCESSED") {
 				// Print seconds as a time value.
-				row.push_back(prettyAge(now - query.getResultInt(col)));
+				if (rawFlag) {
+					row.push_back(format("%d", query.getResultInt(col)));
```

Comment:
It depends on how you interpret what the raw flag should do. The description says it's spitting out DB values, but I think giving the deltas is acceptable and would eliminate possible issues with unsynchronized clocks.